tests/v2/validation/fleet/snapshot/README.md

```diff
@@ -0,0 +1,39 @@
+# Snapshot
+
+For the snapshot tests, these tests are designed to accept an existing cluster that the user has access to. If you do not have a downstream cluster in Rancher, you should create one first before running these tests. It is recommended to have a cluster configuration of 3 etcd, 2 controlplane, 3 workers.
+
+Please see below for more details for your config. Please note that the config can be in either JSON or YAML (all examples are illustrated in YAML).
+
+## Table of Contents
+1. [Getting Started](#Getting-Started)
+
+## Getting Started
+In your config file, set the following:
+```yaml
+rancher:
+  host: "rancher_server_address"
+  adminToken: "rancher_admin_token"
+  clusterName: "cluster_to_run_tests_on"
+  insecure: true/optional
+  cleanup: false/optional
+snapshotInput:
+  upgradeKubernetesVersion: ""        # If left blank, the default version in Rancher will be used.
+  snapshotRestore: "all"              # Options include none, kubernetesVersion, all. Option 'none' means that only the etcd will be restored.
+  controlPlaneConcurrencyValue: "15%"
+  workerConcurrencyValue: "20%"
+  controlPlaneUnavailableValue: "1"
+  workerUnavailableValue: "10%"
```

Comment:
aren't these values optional or they need to be set by the user is it?

Reply:
I don't have a dynamic test so this can be removed, as I hard coded values to make it simpler to run. I don't see value in having a dynamic test for this use case.